smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/DirectedSwiftCodegen.kt

```diff
@@ -107,6 +107,9 @@ class DirectedSwiftCodegen(
                 DependencyJSONGenerator(ctx).writePackageJSON(writers.dependencies)
             }
 
+            LOGGER.info("Generating Smithy model file info")
+            SmithyModelFileInfoGenerator(ctx).writeSmithyModelFileInfo()
```

Comment:
Invokes the model file info generator (see generator implementation immediately below.)